src/views/workflow-summary-tab/config/workflow-summary-tab-details.config.ts

```diff
@@ -0,0 +1,121 @@
+import { createElement } from 'react';
+
+import { StyledLink } from 'baseui/link';
+
+import WorkflowStatusTag from '@/views/shared/workflow-status-tag/workflow-status-tag';
+import getWorkflowIsCompleted from '@/views/workflow-page/helpers/get-workflow-is-completed';
+import getWorkflowStatusTagProps from '@/views/workflow-page/helpers/get-workflow-status-tag-props';
+
+import getWorkflowExecutionLinkProps from '../helpers/get-workflow-execution-link-props';
+import { type WorkflowSummaryTabDetailsConfig } from '../workflow-summary-tab-details/workflow-summary-tab-details.types';
+
+const workflowSummaryTabDetailsConfig: WorkflowSummaryTabDetailsConfig[] = [
+  {
+    key: 'status',
+    getLabel: () => 'Status',
+    valueComponent: ({ lastEvent, params }) =>
+      createElement(
```

Comment:
Is there a reason we don't want to use .tsx for config files? I think doing that here instead of using React.createElement would be much more readable